components/backup/base/privatelink/backup-toolchain-host-schedule.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: velero.io/v1
```

Comment:
This one looks very similar to the schedule we use on public clusters, can we reuse the same one?